core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/operators/OperatorDescriptor.scala

```diff
@@ -70,6 +70,9 @@ import java.util.UUID
 )
 abstract class OperatorDescriptor extends Serializable {
 
+  @JsonIgnore
+  var cachedOutputSchema: Schema = _
```

Comment:
why call it cachedOutputSchema? why not just outputSchema?

Reply:
Actually the reason is quite trivial. There already is a `getOutputSchema` function. So, if I called it `outputSchema`, then I couldn't call the getter as `getOutputSchema`. Open to better names though.